docs/development/install.rst

```diff
@@ -84,11 +85,34 @@ Check that everything works
 #. click on the "View docs" button to browse the documentation, and verify that it works
 
 
+.. admonition:: Front End Development
+
+  If you were sent to this page from the *Front End Development Getting Started* section, you can now `go back and follow the instructions there </development/front-end.html#getting-started>`_.
+
+
+
+
 Working with Docker Compose
 ---------------------------
 
-We wrote a wrapper with ``invoke`` around ``docker-compose`` to have some shortcuts and
-save some work while typing docker compose commands. This section explains these ``invoke`` commands:
+After the initial setup you'll need to run a few commands each time.
+
+#. get the containers running:
+
+  .. prompt:: bash
+
+    inv docker.up  # Options: --no-search --no-reload
+
+#. do stuff
```

Comment:
I may call this section "Normal development workflow" or similar. Something that communicates that this is for day-to-day work.

Also, it can be expanded a little more what to do after "do stuff". I don't remember if the fontend guide explains how to check the JS/CSS changes; in that case we could like there.

On the other hand, if the change is for the backend, we could mention that the services are auto-reloaded unless `--no-reload` was passed, and in that case the services should be restarted manually (could link to `inv docker.restart <service>` or `inv docker.down` and `inv docker.up` again).

Reply:
Made some changes:

* Added the title _"Common development workflow"_ and made this a separate section from the next _"Working with Docker Compose"_.
* Added more info on the `--no-reload` and restarting process, first a short note inside the Warning on the _"Common development workflow"_ section and a bit more expanded bellow on the command list, under the `--no-reload` flag description.


I would also love to expand on the _"do stuff"_. Any suggestions? 👼